components/rmrk/Gallery/GalleryItemCarousel.vue

```diff
@@ -48,15 +48,13 @@ export default class GalleryItemCarousel extends Vue {
       })
 
       const nftsWithCollectionId = data?.collectionEntityById?.nfts.map(
-        (nft) => {
-          return {
-            collection: {
-              id: data.collectionEntityById.id,
-              name: data.collectionEntityById.name,
-            },
-            ...nft,
-          }
-        }
+        (nft) => ({
+          collection: {
+            id: data.collectionEntityById.id,
```

Comment:
@roiLeo use GraphQL aliasing ;)

Reply:
@vikiival `collection: data.collectionEntityById,` ? or do you mean something else?

Reply:
```graphql
query MyQuery($id: ID!) {
  collectionEntityById(id: $id) {
    id
    name
  }
}
```
result

```json
{
  "data": {
    "collectionEntityById": {
      "id": "7472058104f9f93924-KSMRAI",
      "name": "Kusamurais"
    }
  }
}
```

with alias

```graphql
query MyQuery($id: ID!) {
  collection: collectionEntityById(id: $id) {
    id
    name
  }
}
```
result

```json
{
  "data": {
    "collection": {
      "id": "7472058104f9f93924-KSMRAI",
      "name": "Kusamurais"
    }
  }
}
```